app/views/hyrax/dashboard/collections/_show_document_list_row.html.erb

```diff
@@ -20,6 +20,7 @@
     </div>
   </td>
   <td class="text-center"><%= document.date_uploaded %> </td>
+  <td class="text-center">[Owner Goes Here]</td>
```

Comment:
Does this need to be completed?

Reply:
@elrayle suggested creating a new ticket to get this in place.  https://github.com/samvera/hyrax/issues/2478